homeassistant/components/notify/discord.py

```diff
@@ -53,9 +55,15 @@ def async_send_message(self, message, **kwargs):
         def on_ready():
             """Send the messages when the bot is ready."""
             try:
+                data = kwargs.get(ATTR_DATA)
+                if data:
+                    images=data.get(ATTR_IMAGES)
                 for channelid in kwargs[ATTR_TARGET]:
                     channel = discord.Object(id=channelid)
                     yield from discord_bot.send_message(channel, message)
+                    if data:
```

Comment:
Isn't this already checked at line 59?

Reply:
I'm guessing we should check if images are truthy here.

Reply:
Fixed, thank you.